ote/src/cljs/ote/ui/table.cljs

```diff
@@ -17,86 +19,88 @@
-            [ui/table-row {:selectable     (boolean on-select)
-                           :selected       (if row-selected? (row-selected? row) false)
-                           :display-border false
+    [ui/table (merge
+                {:wrapperStyle {:overflow "visible"}
+                 :body-style {:padding-right "5px"}
+                 ;; FIXME: When we have tooltips in header labels, body does not need overflow: visible.
+                 ;;        But, if any row includes tooltips, the body must also have visible overflow.
+                 ;;        For now, leaving this commented out.
+                 ;:bodyStyle {:overflow "visible"}
+                 }
+                (when on-select
+                  {:on-row-selection (fn [selected-rows]
+                                       (when (seq selected-rows)
+                                         (let [rows (vec rows)]
+                                           (on-select (map (partial nth rows) selected-rows)))))})
+                (when height
+                  {:height height
+                   :fixed-header true})
+                (when class
+                  {:class class}))
+     [ui/table-header {:key (str random-table-id "-header")
+                       :style {:overflow "visible" :background-color (color :grey300)}
+                       :adjust-for-checkbox false
+                       :display-select-all false}
+      [ui/table-row {:key (str random-table-id "-header-row")
+                     :style {:overflow "visible"}
+                     :selectable false}
+       (doall
+         (map-indexed
+           (fn [i {:keys [name width tooltip tooltip-pos tooltip-len]}]
+             ^{:key (str i "-" name "-header-column")}
+             [ui/table-header-column {:style
+                                      (merge
+                                        (when width
+                                          {:width width})
+                                        {:white-space "pre-wrap"
+                                         :overflow "visible"
+                                         :color (color :grey900)
+                                         :font-size "1em"
+                                         :font-weight "bold"}
+                                        label-style)}
+              (name->label name)
+              (when tooltip
+                [tooltip-icon {:text tooltip :pos tooltip-pos :len tooltip-len}])])
+           headers))]]
+     [ui/table-body {:display-row-checkbox false}
+      (if (empty? rows)
+        [:tr [:td {:colSpan (count headers)}
+              [common/help no-rows-message]]]
+        (doall
+          (map-indexed
+            (fn [i row]
+              ^{:key (if key-fn (key-fn row) (str i "-" random-table-id "-body-row"))}
+              [ui/table-row (merge
+                              (stylefy/use-style (merge {:background-color (if (= 0 (rem i 2)) "#FFFFFF" (color :grey200))} ; Add stripes
```

Comment:
Should colour code be`:gray200` instead of `:grey200` ?

Also, should table use a logical colour symbol like `table-row-color-alt` and `table-row-color`, which are in turn defined to `gray200` or whatever?